lib/ui/pages/coursedetail/screen/album_page.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_app/src/store/picture_storage.dart';
+import 'package:photo_view/photo_view.dart';
+import 'package:get/get.dart';
+import 'dart:io';
+
+class AlbumPage extends StatefulWidget {
+  final String courseId;
+
+  const AlbumPage({Key? key, required this.courseId}) : super(key: key);
+
+  @override
+  State<AlbumPage> createState() => _AlbumPageState();
+}
+
+class _AlbumPageState extends State<AlbumPage> {
+  final _isLoading = true.obs;
+
+  final List<Picture> _pictures = [];
+
+  final PictureStorage _pictureStorage;
+
+  _AlbumPageState() : _pictureStorage = PictureStorage();
+
+  @override
+  void initState() {
+    super.initState();
+    _useCourseIdToGetPicturePaths();
+  }
+
+  void _removePicture(picture) {
+    _pictureStorage.deletePicture(picture);
+    _pictures.remove(picture);
+    setState(() {});
+    Get.back();
+  }
+
+  void _useCourseIdToGetPicturePaths() async {
+    dynamic picturesInfo = await _pictureStorage.getCoursePicture(widget.courseId);
+    for (final pictureInfo in picturesInfo) {
+      int infoId = pictureInfo['_id'];
+      String infoPath = pictureInfo['picturePath'];
+      String infoLabel = pictureInfo['label'];
+      String infoNote = pictureInfo['note'];
```

Comment:
```suggestion
      final int infoId = pictureInfo['_id'];
      final String infoPath = pictureInfo['picturePath'];
      final String infoLabel = pictureInfo['label'];
      final String infoNote = pictureInfo['note'];
```

Suggest adding `final` to all available places.